handlers/Careers/components/CareersContent/index.js

```diff
@@ -38,6 +39,29 @@ var BENEFITS = [
 ];
 
 class CareersContent extends React.Component {
+  constructor() {
+      super();
+      this.state = { activePersonIndex: -1 };
```

Comment:
yeah, right now the component is just one image/popover and accepts whether it's active.. i could further wrap that in an ImagePopoverGrid component